pkg/nfd-worker/nfd-worker.go

```diff
@@ -121,18 +121,19 @@ type ConfigOverrideArgs struct {
 }
 
 type nfdWorker struct {
-	args                Args
-	certWatch           *utils.FsWatcher
-	clientConn          *grpc.ClientConn
-	configFilePath      string
-	config              *NFDConfig
-	kubernetesNamespace string
-	grpcClient          pb.LabelerClient
-	healthServer        *grpc.Server
-	nfdClient           *nfdclient.Clientset
-	stop                chan struct{} // channel for signaling stop
-	featureSources      []source.FeatureSource
-	labelSources        []source.LabelSource
+	args                        Args
+	certWatch                   *utils.FsWatcher
+	clientConn                  *grpc.ClientConn
+	configFilePath              string
+	config                      *NFDConfig
+	kubernetesNamespace         string
+	grpcClient                  pb.LabelerClient
+	healthServer                *grpc.Server
+	nfdClient                   *nfdclient.Clientset
+	stop                        chan struct{} // channel for signaling stop
+	featureSources              []source.FeatureSource
+	numberOfFeatureSourceErrors int
```

Comment:
I think I'll prefer this to be exposed as a metric, but not in the scope of this PR